src/main/java/ReviewResponse2.java

```diff
@@ -0,0 +1,25 @@
+public class ReviewResponse2 {
```

Comment:
The class violates the Single Responsibility Principle (SRP) as it combines data representation and behavior in a single class. (Severity: medium)